modules/javafx.graphics/src/main/java/javafx/scene/Scene.java

```diff
@@ -6506,4 +6521,207 @@ Accessible getAccessible() {
+     * platform-provided preference (see {@link Platform.Preferences}. Likewise, specifying a non-null value for any
+     * given property will override the platform-provided value.
+     *
+     * @see Platform.Preferences
+     * @since 25
+     */
+    public sealed interface Preferences permits ScenePreferences {
+
+        /**
+         * Specifies whether the scene should prefer light text on dark backgrounds, or dark text
+         * on light backgrounds.
+         * <p>
+         * This property corresponds to the following CSS media feature:
+         * <table class="striped">
+         *     <caption>Media Feature</caption>
+         *     <tbody>
+         *         <tr><th>Name</th><td><code>prefers-color-scheme</code></td></tr>
+         *         <tr><th>For</th><td><code>@media</code></td></tr>
+         *         <tr><th>Value</th><td><code>light</code> | <code>dark</code></td></tr>
+         *         <tr><th>Boolean Context</th><td>not applicable</td></tr>
+         *     </tbody>
+         * </table>
+         *
+         * @return the {@code colorScheme} property
+         * @defaultValue {@link Platform.Preferences#getColorScheme()}
+         * @see Platform.Preferences#colorSchemeProperty()
+         */
+        ObjectProperty<ColorScheme> colorSchemeProperty();
+
+        ColorScheme getColorScheme();
+
+        void setColorScheme(ColorScheme colorScheme);
+
+        /**
+         * Specifies whether applications should always show scroll bars. If set to {@code false}, applications
+         * may choose to hide scroll bars that are not actively used, or make them smaller or less noticeable.
+         * <p>
+         * This property corresponds to the following CSS media feature:
+         * <table class="striped">
+         *     <caption>Media Feature</caption>
+         *     <tbody>
+         *         <tr><th>Name</th><td><code>prefers-persistent-scrollbars</code></td></tr>
+         *         <tr><th>For</th><td><code>@media</code></td></tr>
+         *         <tr><th>Value</th><td><code>no-preference</code> | <code>persistent</code></td></tr>
+         *         <tr><th>Boolean Context</th>
+         *             <td><code>no-preference</code> evaluates as <code>false</code></td>
+         *         </tr>
+         *     </tbody>
+         * </table>
+         *
+         * @return the {@code persistentScrollBars} property
+         * @defaultValue {@link Platform.Preferences#isPersistentScrollBars()}
+         * @see Platform.Preferences#persistentScrollBarsProperty()
+         */
+        ObjectProperty<Boolean> persistentScrollBarsProperty();
+
+        /**
+         * Gets the value of the {@code persistentScrollBars} property.
+         *
+         * @return the value of the {@code persistentScrollBars} property
+         * @see #persistentScrollBarsProperty()
+         * @see #setPersistentScrollBars(Boolean)
+         */
+        boolean isPersistentScrollBars();
```

Comment:
do these accessors need individual javadocs?

Reply:
Yes, because the javadoc tool doesn't recognize a method that returns `boolean` as a getter for `ObjectProperty<Boolean>` (it expects `BooleanProperty`). This is a null-coalescing property: you can set it to `null`, but it will always evaluate to `true` or `false`, so it doesn't make sense to return `Boolean` from its getter.